frontend/src/operators/StoryTreeOperator.ts

```
@@ -0,0 +1,752 @@
+        if (!quoteCountsFromParent || !quoteCountsFromParent.quoteCounts || quoteCountsFromParent.quoteCounts.length === 0) {
+          console.log(`No quotes found for parentId: ${parentId}, no more levels to load, level ${levelNumber} is last level`);
+          this.dispatchLastLevel(levelNumber);
+          break;
+        }
+      }
+      try {
+        let selectedQuoteFromParent = this.fullQuoteFromText(parentText, parentId);
+        if (parentLevel.isLastLevel || !parentLevel.midLevel ){
+          throw new StoryTreeError(`Level ${levelNumber} is the last level, but shouldnt be?`);
+        } else {
+          if (!!parentLevel.midLevel.selectedQuote) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)